src/etcd_client.rs

```diff
@@ -146,6 +156,22 @@ impl EtcdClient {
         }
     }
 
+    /// Write key value pair to etcd, if key exists, update it
+    pub fn write_kv<T: DeserializeOwned + Serialize + Clone + Debug + Send + Sync>(
```

Comment:
change function name to `write_or_update_kv`

Reply:
Done